src/engine/strat_engine/engine.rs

```diff
@@ -253,6 +253,65 @@ impl Engine for StratEngine {
         }
     }
 
+    fn unlock_all(&mut self) -> SetUnlockAction<DevUuid> {
+        fn handle_luks(luks_info: &LLuksInfo) -> bool {
+            if let Ok(Some(mut handle)) = CryptHandle::setup(&luks_info.ids.devnode) {
+                if let Err(e) = handle.activate() {
+                    debug!(
+                        "Failed to open device with path {}, UUID {}. This is \
+                        probably due to a missing key: {}",
+                        luks_info.ids.devnode.display(),
+                        luks_info.ids.identifiers.device_uuid.to_simple_ref(),
+                        e,
+                    );
+                    false
+                } else {
+                    true
+                }
+            } else {
```

Comment:
One note here is that we may not want to report this as "still locked" as this case would be hit more due to disk formatting issues. What do you think, @mulkieran?